sdk/nodejs/policy/server.ts

```diff
@@ -157,7 +193,12 @@ function makeAnalyzeRpcFun(
         const ds: Diagnostic[] = [];
         try {
             for (const p of policies) {
-                const enforcementLevel = p.enforcementLevel || policyPackEnforcementLevel;
+                // If there is a configured enforcement level for the policy, it is used. Otherwise, the enforcement
```

Comment:
Im not quite sure what this comment is saying! (as well as below on line 323, which appears to be the same comment)

Reply:
Decide to just remove the comment instead of rewording. From the code, it should be clear where we're getting the enforcementLevel value.